drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/rebel.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.zeroturnaround.com" xsi:schemaLocation="http://www.zeroturnaround.com http://www.zeroturnaround.com/alderaan/rebel-2_0.xsd">
+
+	<classpath>
+		<dir name="/home/rikkola/Work/Codes/master/drools-wb/drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/target/classes">
```

Comment:
Absolute path. Could it be avoided by using `${project.build.outputDirectory}` property coming from [Super POM](http://maven.apache.org/guides/introduction/introduction-to-the-pom.html#Super_POM) and enabling [resource filtering](http://maven.apache.org/plugins/maven-resources-plugin/examples/filter.html)?